### PR TITLE
Implement granicus api crawler

### PR DESCRIPTION
**Pull request by davidrideout**
* Implemented web crawler to update firestore with granicus data. Unfortunately we need higher tier access to run it as a cronjob.
* Fixed eslint settings for /functions and /web
* Did a really janky implementation for Agenda items until we can discuss further (see slack)

### CODE REVIEW

File: web/src/views/home/AgendaDetailView/AgendaPanel.tsx [hunk 1]
```diff
@@ -28,12 +34,29 @@ function AgendaPanel({ agendaItem }: AgendaPanelProps){
...
-      >
-        <span dangerouslySetInnerHTML={{ __html: agendaItem.description }} />
-      </Typography>
+      <pre className={classes.preformatted}>
+        {agendaItem.description.trim()}
+      </pre>
+
+      <Typography variant={"h3"} className={classes.header}>Location</Typography>
+      <pre className={classes.preformatted}>
+        <MuiLinkify>{agendaItem.location.trim()}</MuiLinkify>
```
Comment (davidrideout) [inline web/src/views/home/AgendaDetailView/AgendaPanel.tsx:42]: Move `trim()` to crawler.

File: functions/package.json [hunk 2]
```diff
@@ -1,26 +1,41 @@
...
+    "logs": "firebase functions:log",
+    "crawl": "functions-framework --target=helloWorld"
   },
   "engines": {
-    "node": "8"
+    "node": "12"
   },
-  "main": "lib/index.js",
+  "main": "lib/functions/src/index.js",
   "dependencies": {
```
Comment (johnkmzhou) [inline functions/package.json:17]: You don't need some of these dependencies.

**Review by johnkmzhou — CHANGES_REQUESTED** [review]
Just a minor change request.

File: functions/package.json (hunk 2 above)
Comment (davidrideout) [reply functions/package.json:17]: Removed unnecessary dependencies

File: web/src/views/home/AgendaDetailView/AgendaPanel.tsx (hunk 1 above)
Comment (davidrideout) [reply web/src/views/home/AgendaDetailView/AgendaPanel.tsx:42]: fixed

**Review by johnkmzhou — APPROVED** [review]
lgtm

**Pull request merged**